.github/workflows/publish.yml

```diff
@@ -0,0 +1,80 @@
+name: Publish and deploy
+
+# Note: Adapted from terra-data-catalog.
+# For future functionality like 'publish to Artifactory', 'push to GCR', ..., see the data catalog version of this file.
+
+# 'create' means upon branch or tag creation, so the completion of the workflow
+# in tag.yml - which creates a tag - implicitly kicks this one off.
+# workflow_dispatch is for manual kickoff with a non-standard tag.
+on:
+  create:
+  workflow_dispatch:
+    inputs:
+      tagOverride:
+        description: 'Specify the tag to use for this publish'
+        required: true
+
+env:
+  SERVICE_NAME: cbas-ui
+  DOCKERHUB_ORG: broadinstitute
+  GOOGLE_PROJECT: broad-dsp-gcr-public
+
+jobs:
+  - publish-job:
+      if: startsWith(github.ref, 'refs/tags/') || github.event.inputs.tagOverride != ''
+      runs-on: ubuntu-latest
+      permissions:
+        contents: 'read'
+        id-token: 'write'
+
+      steps:
+        - uses: actions/checkout@v2
+
+        - name: Parse tag
+          # Sets up a step output called TAG with the appropriate tag
+          id: tag
+          run: |
+            TAG_OVERRIDE=${{ github.event.inputs.tagOverride }}
+            if [[ ! -z "${TAG_OVERRIDE}" ]]
+            then
+              echo ::set-output name=TAG::${TAG_OVERRIDE}
+            else
+              echo ::set-output name=TAG::$(git describe --tags)
+            fi
+
+        - name: Construct dockerhub and GCR image names
+          id: image-name
+          run: |
+            echo ::set-output name=DOCKERHUB_NAME::${DOCKERHUB_ORG}/${SERVICE_NAME}:${{ steps.tag.outputs.TAG }}
+            echo ::set-output name=GCR_NAME::us.gcr.io/${GOOGLE_PROJECT}/${SERVICE_NAME}:${{ steps.tag.outputs.TAG }}
+
+        - name: Build Docker image
+          run: docker build . --no-cache -t ${{ steps.image-name.outputs.DOCKERHUB_NAME }}
+
+        - name: Login to Docker Hub
+          uses: docker/login-action@v1
+          with:
+            username: dsdejenkins
+            password: ${{ secrets.DSDEJENKINS_DOCKERHUB_PASSWORD }}
+
+        - name: Push dockerhub image
```

Comment:
The instructions in this repo's readme, which describe building an image and pushing it to GCR, don't mention dockerhub. Should I take these dockerhub steps out?

Reply:
(the answer was yes)